tpls/mdspan/include/experimental/__p2642_bits/layout_padded.hpp

```diff
@@ -95,6 +95,7 @@ struct padded_extent {
   using static_array_type = typename static_array_type_for_padded_extent<
       padding_value, _Extents, _ExtentToPadIdx, _Extents::rank()>::type;
 
+  MDSPAN_INLINE_FUNCTION
```

Comment:
`inline` is implicitly implied I think. Would be good to remove that useless specifier if so.

Reply:
See https://github.com/kokkos/mdspan/pull/358/files#r1742863263.

Reply:
My 2 cents is that doing things "for code style consistency" is a bad practice IMO. In this case, stripping out `INLINE` makes the code shorter and more to the point, and remove a clearly useless specifier.

But do whatever you want :smile: